README.vi.md

```diff
@@ -0,0 +1,16 @@
+<img width="150" height="150" align="left" style="float: left; margin: 0 10px 0 0;" alt="MS-DOS logo" src="https://github.com/Microsoft/MS-DOS/blob/master/msdos-logo.png">
+
+# Mã nguồn MS-DOS v1.25 và v2.0
+Repo này chứa mã nguồn và chương trình đã biên dịch của MS-DOS v1.25 và MS-DOS v2.0.
+
+Đây là những tệp tương tự được chia sẻ lần đầu tại [Bảo tàng Lịch sử Máy tính vào ngày 25 tháng 3 năm 2014]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) và được tái xuất bản trong repo này để giúp mọi người dễ dàng tìm thấy, tham khảo bằng văn bản và công trình bên ngoài và cho phép khám phá và thử nghiệm những người quan tâm đến các hệ điều hành PC đầu tiên.
+
+# Giấy phép
+Tất cả các tệp trong repo này được phát hành theo [Giấy phép MIT (OSI)]( https://en.wikipedia.org/wiki/MIT_License) theo tệp [LICENSE](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) được lưu trữ trong thư mục gốc của repo này.
+
+# Đóng góp!
+Các tệp nguồn trong repo này là tham chiếu lịch sử và sẽ được giữ nguyên, vì vậy, đừng gửi bất kỷ PR nào sửa đổi mã nguồn, nhưng bạn có thể thoải mái fork và thử nghiệm nó 😊.
```

Comment:
I prefer replacing "tập tin" by "tệp", "repo" by "mã nguồn", "PR" by "Pull Request", "fork" by "sao chép". On the other the hand, there is a typo "bất kỷ", it should be "bất kỳ".

Reply:
I don't use the word "Tập tin",

"Repo, PR, Fork" are common words and no word in Vietnamese fit to this meaning

Reply:
It looks good now. Thank you for your update. 👍